### ISSUE TITLE
Set Base constants

### ISSUE THREAD

**Issue by jordanbancino** [Team6090/FRC6090-2020#1]
Update DriveTrain.java constants to actual values

```java
  public static final double BASE_WIDTH = 10;
  public static final double BASE_LENGTH = 10;
```

**Comment by jordanbancino** [comment]
I'll hopefully take care of this if I can find time, since I've got the bot this weekend.

**Comment by jordanbancino** [comment]
Fixed in [3980ae02f257984f7a00bc2f79decc060863adac](https://github.com/Team6090/FRC6090-2020/commit/3980ae02f257984f7a00bc2f79decc060863adac).
